tox.ini

```diff
@@ -1,21 +1,20 @@
 [gh-actions]
 python =
-    3.8: py38
-    3.9: py39
+    3.9: py39, lint
```

Comment:
I usually prefer linting on the newest version, can we move it to 3.13 instead?

Reply:
Yes, no problem.

Reply:
Done -> linting with Python 3.13 (the same version for the `basepython` variable - is it OK?).

But there is a problem -> https://github.com/Tinche/aiofiles/actions/runs/13092514351/job/36530826666?pr=204 . I am new with `tox`, so I need time to walk through the docs.

If you know the solution, I would be glad to know it as well and I'd fix the problem quick.

Reply:
I think tests/test_os.py just needs to be reformatted with Black on 3.13?

I would have expected Black to behave the same regardless of Python version, but maybe not.

Reply:
Hm no, it's flake8 that's complaining.

Reply:
Ok, maybe updating black or flake8 (or both) may help, I'll look at their changelogs especially for Python 3.13, but a bit later. Locally all checks (lint and test) under Python 3.9 went smoothly.

Reply:
I updated flake8, let's see if it helps.

Reply:
Yes, it worked gracefully, thank you.